src/Http/Controllers/GraphqlController.php

```diff
@@ -16,21 +16,32 @@
 
 class GraphqlController extends Controller {
 
-	protected Application $app;
+	protected IPackages $packages;
 
-	public function __construct(Application $app)
+	public function __construct(IPackages $packages)
 	{
-		$this->app = $app;
+		$this->packages = $packages;
 	}
 
+	/**
+	 * @return \Illuminate\Http\Response|\Illuminate\Http\JsonResponse|\Laravel\Lumen\Http\ResponseFactory|mixed
```

Comment:
this equals to (BS|BS|BS|true) that always is true

Reply:
I changed it to mixed